localization/autoware_pose_covariance_modifier_node/README.md

```diff
@@ -0,0 +1,61 @@
+# Overview
+
+This package is used to enable GNSS and NDT to be used together in localization.
+GNSS and NDT are different sources that generate poses for use in EKF. For EKF,
+the covariance values of the pose source are important. The GNSS system can provide
+us with its own error (RMSE) values. Using these error values, covariance values for
+GNSS can be calculated. However, in default autoware, NDT covariance values are determined
+by default values. While this package manages which pose source will be used by reference
+to the error values coming from GNSS, it also manages situations where GNSS and NDT are used together.
+
+## Flowchart
+
+- Default Autoware Pose Source is only NDT:
+
+<p align="center">
+<img src="./media/new_proposal-original.drawio.png" width="320">
+</p>
+
+- You can see how autoware_pose_covariance_modifier_node works in the diagram below:
+
+<p align="center">
+<img src="./media/new_proposal-proposal.drawio.png" width="620">
+</p>
+
+## Activate This Feuture
```

Comment:
```suggestion
## Activate This Feature
```

Could you go over this and other similar spell check mistakes?

![brave_9qwb3NsXVn](https://github.com/autowarefoundation/autoware.universe/assets/10751153/9428c24c-9f38-4d4a-8b42-b8e791eb8042)

![brave_Gi6M32R0eN](https://github.com/autowarefoundation/autoware.universe/assets/10751153/9e88f221-7766-4669-8ce8-40cf683db804)